CHANGELOG.md

```diff
@@ -25,6 +27,14 @@ which it is deprecated.
   type is deprecated and will be removed in a future release. Set `ccmParams.additionalData` instead.
 - `SwapSDK.prototype.executeSwap`: The `ccmParams.cfParameters` property of the `ExecuteSwapParams`
   type is deprecated and will be removed in a future release. Set `ccmParams.additionalData` instead.
+- `SwapSDK.prototype.executeSwap` and `SwapSDK.prototype.approveAndExecuteSwap`: The methods were deprecated
+  because they are compatible with EVM chains and the ethers.js library only. As a replacement, you should
+  use the new `SwapSDK.prototype.encodeVaultSwapData` method to get the unsigned transaction data for a swap.
+  This data can be signed and submitted to the source chain with your preferred web3 library (eg ethers.js or viem).
+  Examples are available in the Chainflip SDK documentation: https://docs.chainflip.io/swapping/integrations/javascript-sdk/introduction
+- `SwapSDK.prototype.approveVault`: The method was deprecated because it is compatible the ethers.js library only.
+  Use your preferred web3 library (eg ethers.js or viem) to approve the vault address and source token returned from
```

Comment:
```suggestion
- `SwapSDK.prototype.approveVault`: The method was deprecated because it is only compatible with the ethers.js library.
  Use your preferred web3 library (eg ethers.js or viem) to approve the vault address and source token returned from
```